umm-spec-lib/src/cmr/umm_spec/test/expected_conversion_util.clj

```diff
@@ -222,3 +223,15 @@
                  (update pr :OnlineResource dif-online-resource)
                  pr))
       check-nil-pub-ref))
+
+(defn expand-file-dist-infos
+  "Parse each format to see if the format contains multiple values. If it does
+   then replace the distribution info that exists and add new ones for each
+   multple format that exists."
+  [file-dist-info]
+  (let [formats (get (select-keys file-dist-info [:Format]) :Format)
+        parsed-format (format-util/parse-distribution-formats formats)]
```

Comment:
I'm not sure what you are trying to do here, but `for` is a list comprehension, so you are going to get a list back here. It seems like you are trying to mutate `file-dist-info` using a `for loop`, which is not what this does. If you _are_ trying to get a list, I don't think the `conj` on line 237 is doing anything since you are only giving it one argument.

Reply:
Also, if you are trying to create a list here, you don't need a `for` _and_ `map-indexed`. Just call `map-indexed` with a function that takes the index and value and do whatever you want with them in there.

Reply:
You are correct, I don't need conj and therefore I don't need to know the index anymore.  I was able to reduce this function to the following:

(let [formats (:Format (assoc file-dist-info :Format "try, this"))
      parsed-format (format-util/parse-distribution-formats formats)]
  (map #(assoc file-dist-info :Format %) parsed-format))